app/models/manageiq/providers/vmware/infra_manager/inventory/persister.rb

```diff
@@ -3,7 +3,12 @@ class ManageIQ::Providers::Vmware::InfraManager::Inventory::Persister < ManageIQ
   require_nested :Full
   require_nested :Targeted
 
+  attr_reader :uuid
```

Comment:
Naming is hard - wondering if this should this be more specifically named... `tracking_uuid`, `tracking_label`, something like that?

Reply:
Yeah it is :) tracking_uuid sounds good

Reply:
Done